Documentation/ceph-osd-mgmt.md

```diff
@@ -81,6 +81,9 @@ If you want to reduce the storage in your cluster, a healthy PVC-based OSD can b
    - `kubectl -n rook-ceph get pod -l ceph.rook.io/pvc=<orphaned-pvc> -o yaml | grep ceph-osd-id`
    - For example, this might return: `ceph-osd-id: "0"`
 
+If you later increase the count in the device set, note that the operator will create PVCs with a new index rather than using the
+same PVC names that were removed in these steps.
```

Comment:
In two OSDs (OSD0 and OSD1) environment, I confirmed that OSD1's PVC name became `rook-ceph-osd-set1-data-1` after removing OSD1 and adding it again. It's probably because the index is the highest existing number + 1. So it's not always incremented.

Reply:
Yes, I'll clarify. If the highest index is deleted, it will be re-used.